src/lecturer/components/pickQuizView/PickQuizView.tsx

```diff
@@ -107,16 +111,27 @@ export function PickQuizView() {
             maxWidth: "100%",
         },
     })();
-    const array = setArray(questionListMock.length);
+    const [indexArray, setIndexArray] = useState<number[]>([]);
     const [checked, setChecked] = useState<number[]>([]);
-    const [left, setLeft] = useState<number[]>(array);
+    const [left, setLeft] = useState<number[]>(indexArray);
     const [right, setRight] = useState<number[]>([]);
-    const [quizes, setQuizes] = useState<Quiz[]>([]);
-    const [name, setName] = useState("");
+    const [title, setTitle] = useState("");
     const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
-        setName(event.target.value);
+        setTitle(event.target.value);
     };
-    const [question, setQuestion] = useState<Question[]>(questionListMock);
+    const [questions, setQuestions] = useState<Question[]>([]);
+
+
+    useEffect(() => {
```

Comment:
why there are two functions with exactly the same name in one file?

Reply:
useEffect  is react hook 
https://pl.reactjs.org/docs/hooks-effect.html